sms_analysis.ipynb

```diff
@@ -605,28 +698,36 @@
    "execution_count": null,
    "metadata": {
     "collapsed": false,
+    "deletable": true,
+    "editable": true,
     "scrolled": false
    },
    "outputs": [],
    "source": [
     "widgets.interact(\n",
     "    get_word_summary_for_diffs,\n",
-    "    contact=widgets.Text(description='1st Contact:', placeholder='Enter 1st name'),\n",
-    "    other_contact=widgets.Text(description='2nd Contact:', placeholder='Enter 2nd name'),\n",
+    "    contact=widgets.Text(value='Mom', description='1st Contact:', placeholder='Enter 1st name'),\n",
```

Comment:
Wanted to make it similar to all other widgets, but you're right, this should be a separate PR